### DISCUSSION TITLE
www.travellings.cn不停重定向无法访问

### DISCUSSION THREAD

**Discussion by immccn123** [travellings-link/travellings#1468] [寻求帮助] [answered]
![Screenshot_2023-03-02-22-43-59-613-edit_com android browser](https://user-images.githubusercontent.com/41335471/222461004-c889b91f-2082-4d02-b928-5b39d230c67b.jpg)

注：根域的一部分是好的

**Comment by Big-Cake-jpg** [comment, marked as answer]
已知问题。

由于主页的服务器相关事宜由 @linlinzzo 负责且其学业繁忙，该问题可能将延迟到周末方能进行处理。

在此之前请先通过去除开往外链中的 `www` 来缓解。
